wiki/Submission/zh.md

```diff
@@ -0,0 +1,27 @@
+# 提交
+
+[谱面](/wiki/Beatmaps)可以通过[谱面编辑器](/wiki/Beatmap_Editor)提交至 osu! 网站。提交谱面可以使其他用户注意到该谱面，且该谱面可能会标记为 [Ranked](/wiki/Beatmaps#ranked) 或 [Loved](/wiki/Beatmaps#loved)。
+
+![](img/bss_warning.png "谱面提交系统的截图")
+
+在编辑器内的`文件`下拉列表中选择`上传地图...`（快捷键：`Ctrl` + `Shift` + `U`）会打开**谱面提交系统**（***Beatmap Submission System***，简写为***BSS***）窗口。窗口首先会列出帮助用户解决做图相关的问题、寻求谱面的反馈以及确保地图合理的资源。若在使用系统时遇到问题，请参阅指南[如何解决谱面提交系统的问题](/wiki/Guides/BSS_Issues)。
+
+如果用户上传的谱面还没有提交至 osu! 网站，窗口会提示用户可以再上传多少张地图。若谱面已被提交，窗口会警告上传地图会使谱面重置。若谱面已成为坟图，窗口会警告谱面将会改变为 pending 状态。
+
+## 提交选项
+
+![](img/bss_submitting.png "谱面提交系统上传时的截图")
+
+点击`上传新的地图！`或`更新地图！`按钮后，用户可以选择将他们上传的谱面标记为`Work In Progress/Help`或`Pending Beatmaps`。标记为 WIP 的谱面不能改变排名状态，但 pending 的谱面可以。
```

Comment:
I think you should put spaces around English words when they're in code styling, too
```suggestion
点击`上传新的地图！`或`更新地图！`按钮后，用户可以选择将他们上传的谱面标记为 `Work In Progress/Help` 或 `Pending Beatmaps`。标记为 WIP 的谱面不能改变排名状态，但 pending 的谱面可以。
```